django_loci/channels/receivers.py

```diff
@@ -19,6 +19,23 @@ def update_mobile_location(sender, instance, **kwargs):
         )
 
 
+def update_mobile_all_locations(sender, instance, **kwargs):
+    if not kwargs.get("created") and instance.geometry:
+        channel_layer = channels.layers.get_channel_layer()
+        group_name = "loci.mobile-location.all"
+        message = {
+            "id": str(instance.pk),
+            "geometry": json.loads(instance.geometry.geojson),
+            "address": instance.address,
+            "name": instance.name,
+            "type": instance.type,
+            "is_mobile": instance.is_mobile,
+        }
+        async_to_sync(channel_layer.group_send)(
+            group_name, {"type": "send_message", "message": message}
+        )
```

Comment:
Can we do this operation in the `update_mobile_location` function? 

The function will send message to two different groups

1. Group dedicated to it's ID 
2. Common group for all locations. 

I believe this will lead to more readable code. Just add a docstring and separate the two operations with a comment in between.